python/torch_mlir_e2e_test/test_suite/backprop.py

```diff
@@ -148,6 +181,40 @@ def ConvolutionBackwardModule2DPadded_basic(module, tu: TestUtils):
                        tu.rand(2, 2, 3, 3))
 
 
+class ConvolutionBackwardModule2DStrided(torch.nn.Module):
+
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([1, 2, 4, 4], torch.float32, True),
+        ([1, 2, 8, 8], torch.float32, True),
+        ([2, 2, 3, 3], torch.float32, True),
+    ])
+    def forward(self, grad_out, input_vec, weight):
+        return torch.ops.aten.convolution_backward(
+            grad_out,
+            input_vec,
+            weight,
+            bias_sizes=[4],
+            stride=[2, 2],
+            padding=[1, 1],
+            dilation=[1, 1],
+            transposed=False,
+            output_padding=[0, 0],
```

Comment:
This implementation only supports non-zero output padding. Here's the existing check: https://github.com/llvm/torch-mlir/pull/1773/files#diff-bec402e56f7c88a9b22018c90df7e9ca721bdd097d4ffdd7c13d9d3f4c2e0688R1685-R1692

Reply:
I think my comment was a bit confusing. I meant adding a test for `AtenConvolutionOp`, since the changes to `Linear.cpp` have expanded the ops support to handle non-zero `output_padding`